.gitignore

```diff
@@ -16,3 +16,4 @@ tests/metadata
 .vscode/
 .idea/
 
+tests/priority/*
```

Comment:
Why add this to `.gitingore`?

Reply:
I didn't.  This must have come in from a merge from the main branch.

Reply:
What if in the future people need to add some test scripts under `tests/priority`? 
If the json files are temporary, they are not supposed to be generated to a general purpose folder.

Reply:
I did not setup the writing of these json files.  I saw them and they did not look appropriate to check in to a repo.  I can remove it from ignore, but the code that is writing these files should be modified so it isn't writing temp files to a test directory.

Reply:
@wangxin, as per the comment, it seems json files are coming from our test infra?
the PR is fine now with the latest changes(as per the functionality).. Let me know if we should go ahead with merge or wait for the json file resolution?

Reply:
If `tests/priority/*` is not related with this change, it would be better to remove them from `.gitingore`.
With this entry in `.gitignore`, this can be interpreted like this: "Files `tests/priority/*` must not be included in git for the BGP session flap tests". It doesn't make sense, right?

Reply:
I removed this line from the ignore file.